tests/testthat/test-module_teal.R

```diff
@@ -2974,3 +3034,118 @@ testthat::describe("teal-reporter", {
     )
   })
 })
+
+testthat::describe("teal-src", {
+  it("Show R code button contains 'is available' reason when module's server returns NULL", {
```

Comment:
there is a huge difference between those two meanings
```suggestion
  it("Show R code button contains 'no code available' reason when module's server returns NULL", {
```